go.mod

```diff
@@ -66,3 +66,5 @@ require (
 	google.golang.org/grpc v1.60.0 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
 )
+
+replace github.com/vmware/go-vcloud-director/v2 => github.com/Didainius/go-vcloud-director/v2 v2.17.0-alpha.2.0.20240426055454-19d6098e3f6b
```

Comment:
reminder to remove this replacement before merging

Reply:
Done.